src/components/common/ClassroomEditor.jsx

```diff
@@ -0,0 +1,240 @@
+          <Layer closer={true} onClose={props.toggleFormVisibility}>
+            <Toast status="critical">
+              { /* //TODO */ }
+              WARNING: Don't actually submit anything, this is a TODO. Clicking will create a new Classroom, not edit an existing one!
+            </Toast>
+            <ClassroomCreateFormContainer />
+          </Layer>
+        }
+        
+        <Box
+          align="center"
+          direction="row"
+          justify="between"
+          pad="small"
+        >
+          <Box flex={true}>
+            <Paragraph align="start" size="small">Click a project column to toggle between percentages and number of classifications.</Paragraph>
+          </Box>
+          
+          <CopyToClipboard className="join-link" text={joinURL} onCopy={props.copyJoinLink}>
+            <Button type="button" label="Copy Join Link"></Button>
+          </CopyToClipboard>
+          
+          <Button type="button" primary={true} label="Export Grades" onClick={this.exportGrades} />
+        </Box>
+        
+        <Box
+          className="manager-summary"
+          pad="small"
+        >
+          <Box pad="small">
+            <Anchor
+              onClick={()=>{console.log(props.selectClassroom); props.selectClassroom(null)}}
+              icon={<LinkPreviousIcon size="small" />}
+              label="Back to Classroom List"
+              className="summary__return-link"
+            />
+          </Box>
+          <Box direction="row">
+            <Button
+              className="manager-table__button--edit"
+              type="button"
+              onClick={props.toggleFormVisibility}
+              icon={<EditIcon size="small" />}
+            ></Button>
+            <Heading tag="h2" strong={true}>{props.selectedClassroom.name}</Heading>
+          </Box>
+          <List>
+            {(!(props.selectedClassroom.subject && props.selectedClassroom.subject.length > 0)) ? null :
+              <ListItem>
+                <Label className="secondary">Subject</Label>
+                <span>{props.selectedClassroom.subject}</span>
+              </ListItem>
+            }
+            {(!(props.selectedClassroom.school && props.selectedClassroom.school.length > 0)) ? null :
+              <ListItem>
+                <Label className="secondary">Institution</Label>
+                <span>{props.selectedClassroom.school}</span>
+              </ListItem>
+            }
+            {(!(props.selectedClassroom.description && props.selectedClassroom.description.length > 0)) ? null :
+              <ListItem>
+                <Label className="secondary">Description</Label>
+                <span>{props.selectedClassroom.description}</span>
+              </ListItem>
+            }
+            <ListItem>
+              <Label className="secondary">Join Link</Label>
+              <span>{joinURL}</span>
+            </ListItem>
+          </List>
+        </Box>
+
+        <Table className="manager-table">
```

Comment:
The `<table>` structure for the `.classroom-editor` is a much, much more stripped down version of the `.classroom-manager`; seems to work, but please let me know if I'm missing something crucial.